src/style.css

```diff
@@ -1,3 +1,90 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
You should add fallback fonts. This is a code quality issue because if the Roboto font fails to load for some reason, there won't be a backup font to display. You can use a generic font family as a fallback, like 'sans-serif'.